code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -169,6 +169,10 @@
 			if(internals)
 				internals.icon_state = "internal0"
 			internal = null
+		if(istype(I, /obj/item/clothing/mask/muzzle/tapegag))//snowflakey tape gag. Literally no other place to put this.

```

Comment:
you should use `dropped` for this. You can specifically check for if they're equipped or not there, and it'll work---that's now the noir glasses handle things: https://github.com/ParadiseSS13/Paradise/blob/e34fa95fae2d7b63cc7fa3232970c0b5452415de/code/modules/clothing/glasses/glasses.dm#L247